code/src/Components/Wrapper.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+
+const Wrapper = ({ children }) => {
+  return (
+    <OuterWrapper>
+      <InnerWrapper>{children}</InnerWrapper>
+    </OuterWrapper>
+  );
+}
+
+export default Wrapper;
+const OuterWrapper = styled.div`
+display: grid;
+`
+const InnerWrapper = styled.div`
+min-height: 100vh;
+width: 300px;
+margin: 0 auto;
+justify-content: center;
+text-align: center;
+`
```

Comment:
Maybe you could use these wrappers more in your project, instead of creating more wrappers in the components. Just a thought to think of for future projects.

Reply:
We agree with you! The plan was to reuse the wrapper component it but we discovered along the way that it didn't work since we used a wrapper handling the background images in our game component. So it was a one-off. :)